docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -310,6 +312,20 @@ Uninstall Alluxio as follows:
 $ helm delete alluxio
 ```
 
+#### Format Journal
+
+The helm chart contains one Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+template that can be used for formatting the Alluxio journal. The template contains one Job for each Alluxio master.
+Each Job runs `alluxio formatJournal` and formats the journal for that master.
+
+You can trigger the journal formatting by upgrading the helm deployment with `journal.format.runFormat=true`.
+```console
+# Use the same config.yaml and switch on journal formatting
+$ helm upgrade alluxio --version 0.5.5 -f config.yaml --set journal.format.runFormat=true alluxio-local/alluxio
+```
+
+> Note: `helm upgrade` is another deployment, which will result in new master and worker pods.
```

Comment:
have you tested this? i think helm would run Job after StatefulSet which is not desirable

Reply:
why is ` --version 0.5.5` needed? i plan on removing the helm version tag from documentation

Reply:
The version is removed.